pytorch_lightning/profiler/profiler.py

```diff
@@ -1,11 +1,11 @@
 import cProfile
 import io
-import logging as log
 import pstats
 import time
 from abc import ABC, abstractmethod
 from collections import defaultdict
 from contextlib import contextmanager
+from pytorch_lightning import _logger as log
```

Comment:
the imports are usually structured into three groups, separated by blank line
1. build-in packages (e.g. time)
2. third party packages (e.g numpy)
3. this module - pytorch_lightning

Reply:
Got it!